lib/cartopy/tests/crs/__init__.py

```diff
@@ -20,3 +20,8 @@
 """
 
 from __future__ import (absolute_import, division, print_function)
+
+import pytest
+
+
+pytest.register_assert_rewrite('cartopy.tests.crs.helpers')
```

Comment:
This was a new one for me. Interesting.

Reply:
Yea, I would've put everything in `__init__.py`, but not having assertion rewriting is annoying.